esrally/driver/runner.py

```diff
@@ -2068,6 +2071,62 @@ def __repr__(self, *args, **kwargs):
         return "delete-async-search"
 
 
+class OpenPointInTime(Runner):
+    async def __call__(self, es, params):
+        index = mandatory(params, "index", self)
+        response = es.open_point_in_time(index=index,
+                                         params=params)
+        id = response.get("id")
+        op_name = mandatory(params, "name", self)
+        # is name uniqueness enforced somewhere? this may collide with async searches
+        CompositeContext.put(op_name, id)
+
+    def __repr__(self, *args, **kwargs):
+        return "open-point-in-time"
+
+
+class QueryWithSearchAfterScrolling(Query):
+    async def __call__(self, es, params):
+        body = {"query": mandatory(params, "query", self),
+                "sort": mandatory(params, "sort", self)}
+        size = params.get("results-per-page", 10)
+        body["size"] = size
+        pit_op = params.get("with-point-in-time-from")
+        request_params, headers = self._transport_request_params(params)
+
+        while True:
+            if pit_op:
+                pit_id = CompositeContext.get(pit_op)
+                body["pit"] = {"id": pit_id,
+                               "keep_alive": "1m" }
+            props = await self._raw_search(es, doc_type=None, index=None, body=body.copy(), params=request_params, headers=headers)
```

Comment:
note here we leave eager parsing on and materialize the response.  this is because we need the last hit's sort field in order to know what to search_after